### PR TITLE
Milestone testing

### PR DESCRIPTION
**Pull request by kelvinbush**
In this pull request, we added unit testing for all the components and logic files using `Jest` , `jest snapshots` and `React Testing Library`.

### CODE REVIEW

File: src/logic/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    obj = calculate(obj, '5');
+    obj = calculate(obj, '0');
+    obj = calculate(obj, '0');
+    obj = calculate(obj, '-');
+    obj = calculate(obj, '5');
+    obj = calculate(obj, '0');
+    obj = calculate(obj, '=');
+    expect(obj.total).toEqual('450');
+  });
+});
```
Comment (GzimAsani) [inline src/logic/calculate.test.js:33]: - [ ] Please test all operations include multiply and division

File: src/logic/operate.test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+describe('Operater Test', () => {
+  test('1 + 1 should return 2', () => {
+    const total = operater(1, 1, '+');
+    expect(total).toEqual('2');
+  });
+  test('50 x 10 should return 500', () => {
+    const total = operater(50, 10, 'x');
+    expect(total).toEqual('500');
+  });
+});
```
Comment (GzimAsani) [inline src/logic/operate.test.js]: - [ ] Please implement all the operations like `division` and `substraction`

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hi @kelvinbush  and @jasonfelice    
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job so far :clap: 
 
 ## Required Changes 

- [ ] _Check the comments below :arrow_double_down:_


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
Hi @kelvinbush, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# Highlights
- No linter errors ✔️
- All your test are passing ✔️
- Your code is well organized and clean ✔️
appr


# Status: Approved ✔️✔️✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] To add run test instructions in your readme file as this is testing pull request.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**